### PR TITLE
[plugin.video.embycon] 1.6.18

### PR DESCRIPTION
**Pull request by faush01**
### Description

December Release : 1.6.18

 - add server results caching
 - fix a few last minute Kodi 18 and Emby 3.6 bugs before they release
 - add some more music, home movies and movies nodes to the main menu
 - few bug fixes

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.embycon/resources/lib/functions.py [hunk 1]
```diff
@@ -671,6 +696,11 @@ def show_menu(params):
         li.setProperty('menu_id', 'view_season')
         action_items.append(li)
 
+    if result["Type"] == "Series":
+        li = xbmcgui.ListItem("Goto Series")
```
Comment (enen92) [inline plugin.video.embycon/resources/lib/functions.py]: Can you move this to localized strings?

File: plugin.video.embycon/resources/lib/skin_clone.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+def set_value(name, value):
+    params = {
+        'setting': name,
+        'value': value
+    }
+    result = json_rpc('Settings.setSettingValue').execute(params)
+    log.debug("Save Setting : {0} : {1} : {2} : {3}", name, value, type(value), result)
+
+
+def clone_skin():
```
Comment (enen92) [inline plugin.video.embycon/resources/lib/skin_clone.py]: I don't think addons are allowed to do this. Have to check with others.

Comment (faush01) [reply plugin.video.embycon/resources/lib/skin_clone.py]: are you referring to the clone skin function?

**Comment by faush01** [comment]
the other option is drop from the official repo.
I will run a poll and see what users think.

**Comment by enen92** [comment]
@faush01 whatever you think is best. In case of removal bump the addon version and fill the `broken` tag in addon.xml. I am not completely sure if the addon containing the `clone_skin` code can have support on our forums. If you decide to remove the addon from the repo I think it is better to also share your modded skin in the repo as a dependency for the addon.

**Comment by faush01** [comment]
@enen92 
cool thanks for the info on the deprecation/broken flag.

I dont think the addon has ever even been mentioned on the kodi forums, I cold be wrong here but I dont go there much and when I do it is to not get help with building Kodi from source.

last time I run this poll users though it was better to keep the addon in the official repo and drop the proposed feature so I guess it will probably end up the same this time but is best to ask the user base and other devs I feel.

File: plugin.video.embycon/resources/lib/skin_clone.py (hunk 2 above)
Comment (faush01) [reply plugin.video.embycon/resources/lib/skin_clone.py]: have removed this now

**Comment by faush01** [comment]
@enen92 
remove all the skin cloning
updated to pull string from resource file

**Comment by enen92** [comment]
Thanks much

**Pull request merged**